opm/parser/eclipse/EclipseState/Grid/GridProperties.cpp

```diff
@@ -397,6 +398,118 @@ namespace Opm {
         }
     }
 
+    namespace {
+        /*
+          The functions in this namespace are those listed as
+          available operations in the OPERATE keyword.
+        */
+
+        double MULTA(double, double X, double alpha, double beta) {
+            return alpha*X + beta;
+        }
+
+        // NB: The POLY function and the MULTIPLY function both use
+        // the R value in the calculation. That implies that we should
+        // ideally check that the R property has already been
+        // initialized with a valid value, For all the other
+        // operations R only appears on the left side of the equation,
+        // and can be fully assigned to.
+        double POLY(double R, double X, double alpha, double beta) {
+            return R + alpha * pow(X , beta );
+        }
+
+        double MULTIPLY(double R, double X, double , double ) {
+            return R * X;
+        }
+
+        double SLOG(double, double X, double alpha, double beta) {
+            return pow(10 , alpha + beta * X);
+        }
+
+        double LOG10(double, double X, double , double ) {
+            return log10(X);
+        }
+
+        double LOGE(double, double X, double , double ) {
+            return log(X);
+        }
+
+        double INV(double, double X, double , double ) {
+            return 1.0/X;
+        }
+
+        double MULTX(double, double X, double alpha, double ) {
+            return alpha * X;
+        }
+
+        double ADDX(double, double X, double alpha, double ) {
+            return alpha + X;
+        }
+
+        double COPY(double, double X, double, double ) {
+            return X;
+        }
+
+        double MAXLIM(double, double X, double alpha, double ) {
+            return std::min( alpha , X );
+        }
+
+        double MINLIM(double, double X, double alpha, double ) {
+            return std::max( alpha , X );
```

Comment:
Well - at least according to the documentation - yes; I had a glimpse of understanding while developing this - but that frankly eludes me now.